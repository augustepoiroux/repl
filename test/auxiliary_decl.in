{"cmd": "example : true := by sorry"}

{"tactic": "let p : ∃ x : Nat, x > 0 := ⟨1, Nat.one_pos⟩", "proofState": 0}

{"tactic": "let ⟨n, hn⟩ := p", "proofState": 1}

{"tactic": "rfl", "proofState": 2}

