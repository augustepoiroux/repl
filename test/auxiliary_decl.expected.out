{"sorries":
 [{"proofState": 0,
   "pos": {"line": 1, "column": 21},
   "goal": "⊢ true = true",
   "endPos": {"line": 1, "column": 26}}],
 "messages":
 [{"severity": "warning",
   "pos": {"line": 1, "column": 0},
   "endPos": {"line": 1, "column": 7},
   "data": "declaration uses 'sorry'"}],
 "env": 0}

{"proofStatus": "Incomplete: open goals remain",
 "proofState": 1,
 "goals": ["p : ∃ x, x > 0 := Exists.intro 1 Nat.one_pos\n⊢ true = true"]}

{"proofStatus": "Incomplete: open goals remain",
 "proofState": 2,
 "goals":
 ["p : ∃ x, x > 0 := Exists.intro 1 Nat.one_pos\nn : Nat\nhn : n > 0\n⊢ true = true"]}

{"proofStatus": "Completed", "proofState": 3, "goals": []}

